lib/pilot/tester_manager.rb

```diff
@@ -67,9 +67,19 @@ def list_testers(options)
       start(options)
       require 'terminal-table'
 
-      list(Spaceship::Tunes::Tester::Internal.all, "Internal Testers")
+      app_filter = (config[:apple_id] || config[:app_identifier])
+      if app_filter
+        app = Spaceship::Application.find(app_filter)
+        intTesters=Spaceship::Tunes::Tester::Internal.all_by_app(app.apple_id)
+        extTesters=Spaceship::Tunes::Tester::External.all_by_app(app.apple_id)
+      else
+        intTesters=Spaceship::Tunes::Tester::Internal.all
+        extTesters=Spaceship::Tunes::Tester::External.all
```

Comment:
Surrounding space missing for operator `=`.<br>Use snake_case for variable names.